virttest/utils_test/libvirt.py

```diff
@@ -3303,7 +3282,7 @@ def new_disk_vol_name(pool_name):
         return None
     disk = poolxml.get_source().device_path[5:]
     part_num = len(list(filter(lambda s: s.startswith(disk),
-                               get_parts_list())))
+                               utils_misc.utils_disk.get_parts_list())))
```

Comment:
request to move `new_disk_vol_name()` to utils_disk.py

Reply:
@balamuruhans , I will create other PR for this request.

Reply:
@balamuruhans  It's going on beyond this Pull request. I think we can make a long term plan to reorganize the codes but not in here.  What do you think?

Reply:
Okay @Yingshun, I will take care of that part. I thought it would be easy when we do then and there. Thanks!